### ISSUE TITLE
Add aria-label to language selector

### ISSUE THREAD

**Issue by Bojhan** [TheIdentitySelector/thiss-js#278]
The language selector currently fails WCAG 2.2.

The `#locale-selector ` should have an aria-label which says "Change language"
